builtin/providers/openstack/resource_openstack_dns_recordset_v2.go

```diff
@@ -134,22 +136,25 @@ func resourceDNSRecordSetV2Read(d *schema.ResourceData, meta interface{}) error
 		return fmt.Errorf("Error creating OpenStack DNS client: %s", err)
 	}
 
-	zoneID := d.Get("zone_id").(string)
+	// Obtain relevant info from parsing the ID
+	zoneID, recordsetID, err := parseDNSV2RecordSetID(d.Id())
+	if err != nil {
+		return err
+	}
 
-	n, err := recordsets.Get(dnsClient, zoneID, d.Id()).Extract()
+	n, err := recordsets.Get(dnsClient, zoneID, recordsetID).Extract()
 	if err != nil {
 		return CheckDeleted(d, err, "record_set")
 	}
 
-	log.Printf("[DEBUG] Retrieved  record set %s: %#v", d.Id(), n)
+	log.Printf("[DEBUG] Retrieved  record set %s: %#v", recordsetID, n)
 
 	d.Set("name", n.Name)
 	d.Set("description", n.Description)
 	d.Set("ttl", n.TTL)
 	d.Set("type", n.Type)
 	d.Set("records", n.Records)
 	d.Set("region", GetRegion(d))
-	d.Set("zone_id", n.ZoneID)
```

Comment:
```
=== RUN   TestAccDNSV2RecordSet_importBasic
--- FAIL: TestAccDNSV2RecordSet_importBasic (192.40s)
	testing.go:280: Step 1 error: ImportStateVerify attributes not equivalent. Difference is shown below. Top is actual, bottom is expected.
		
		(map[string]string) {
		}
		
		
		(map[string]string) (len=1) {
		 (string) (len=7) "zone_id": (string) (len=36) "68d14ad5-b21c-4a2a-9208-ca01dfa3677c"
		}
```

Can you see anything obvious that may still be wrong with the `importBasic` test setup?